src/App.tsx

```diff
@@ -127,28 +125,35 @@ export default function App() {
                 includeHighlighting: false,
                 includePrettyResults: false
               });
+              setResults({});
             }}
           >
             Reset
           </Button>
+
           <Button
             variant="contained"
             color="primary"
             onClick={() => {
-              let results = {};
+              const options = {
+                measurementPeriodStart: measurementPeriodStart?.toISOString(),
+                measurementPeriodEnd: measurementPeriodEnd?.toISOString(),
+                ...calculationOptions
+              };
               if (outputType === 'rawResults') {
-                results = Calculator.calculateRaw(measureBundle, [patientBundle], options);
+                setResults(Calculator.calculateRaw(measureBundle, [patientBundle], options));
               } else if (outputType === 'detailedResults') {
-                results = Calculator.calculate(measureBundle, [patientBundle], options);
+                setResults(Calculator.calculate(measureBundle, [patientBundle], options));
               } else if (outputType === 'measureReports') {
-                results = Calculator.calculateMeasureReports(measureBundle, [patientBundle], options);
+                setResults(Calculator.calculateMeasureReports(measureBundle, [patientBundle], options));
               }
-              console.log(results);
             }}
           >
             Calculate
           </Button>
         </Grid>
+        <h2>Results:</h2>
+        <ReactJson src={results} enableClipboard={true} theme="shapeshifter:inverted" collapsed={2} />
```

Comment:
```suggestion
        {results && <ReactJson src={results} enableClipboard={true} theme="shapeshifter:inverted" collapsed={2} />}
```

Here we can only render the json viewer component if results is non-null.